src/tasks/index.ts

```diff
@@ -14,6 +14,12 @@ export class TasksEndpoint extends BaseApi {
     return responseHandlers.list(response);
   }
   public readRaw(requestParams: RequestParams<Task> | PagingParams<Task>): Promise<Response<Task>> {
+    if (!requestParams.getParams) {
+      // Workaround for serialization removing any classes/functions
+      const oldRequestParams = requestParams;
+      requestParams = new RequestParams<Task>();
+      Object.assign(requestParams, oldRequestParams);
```

Comment:
This doesn't look to me like a deep copy. Because oldRequestParams and requestParams will still share the same pointer to the `criteria` object.